mango-cli/src/main/webapp/WEB-INF/templates/refVis.ssp

```diff
@@ -64,6 +64,12 @@ function refVis(dictionary){
     tooltip.style('display', 'block');                          
   });                                                           
   
+  path.on('click', function(d) {
+    var end = 1000;// Show first 1000 , might want to so something more sophisticated here...//d.data.length/100;
```

Comment:
could you view the middle of the chromosome here (ie. data.length/2 + data.length/2 + 1000)? I say this because there is often not data at the beginning of the chromosomes